schema/spacewalk/postgres/triggers/rhnChannelCloned.sql

```diff
@@ -26,7 +26,7 @@ begin
 			( id, channel_id, comps_type_id, relative_filename,
 				last_modified, created, modified )
 		select nextval('rhn_channelcomps_id_seq'), new.id, comps_type_id, relative_filename,
-				current_timestamp, current_timestamp, current_timestamp
+				last_modified, current_timestamp, current_timestamp
```

Comment:
I get the goal, but It will be strange that last_modified can be older than the current_timestamp. For that reason I'm not sure if you should copy the last_modify from the original channel comps.

Reply:
In this table, `last_modified` refers to the last modification date of a comps/modules file in the filesystem. When cloning the row in the DB, the referenced file is not actually touched, so the `last_modified` should stay the same as the original.
Note that there are also generic `created` and `modified` columns that store timestamp information about the DB entities themselves.